ui/src/hooks/vehicle-service/useGetTimetables.ts

```diff
@@ -199,7 +199,7 @@ export interface TimetableWithMetadata {
   journeyPatternId: UUID;
 }
 
-export const useGetTimetables = (journeyPatternId: UUID) => {
+export const useGetTimetables = (journeyPatternId?: UUID) => {
```

Comment:
this is a bad name for a hook. Is this rather a useGetRouteTimetables? (at least the parameter is a journey pattern id, that's why I think so)